crates/claims/crates/vc/src/v2/mod.rs

```diff
@@ -5,9 +5,11 @@ use iref::Iri;
 
 use crate::syntax::RequiredContext;
 
+mod algorithm;
```

Comment:
I don't think the `algorithm` module will contain anything else so I would put it in its own `problem` module.
```rust
pub mod problem;
pub use problem::ProblemDetails;
```